runtime/frequency/src/xcm/parameters.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{
+	polkadot_xcm_fee::default_fee_per_second, Balance, ParachainInfo, RuntimeOrigin,
+	MAXIMUM_BLOCK_WEIGHT,
+};
+use cumulus_primitives_core::AggregateMessageOrigin;
+use frame_support::parameter_types;
+use staging_xcm::latest::{prelude::*, WESTEND_GENESIS_HASH};
+
+pub const MICROUNIT: Balance = 1_000_000;
+pub const RELAY_GENESIS: [u8; 32] = WESTEND_GENESIS_HASH;
+const ASSET_HUB_ID: u32 = 1000;
+
+pub type ForeignAssetsAssetId = Location;
+
+parameter_types! {
+	pub const RelayLocation: Location = Location::parent();
+	pub const RootLocation: Location = Location::here();
+	pub FeeAssetId: AssetId = AssetId(RelayLocation::get());
+	pub const BaseDeliveryFee: u128 = (1_000_000_000_000u128 / 100).saturating_mul(3);
+	pub const TransactionByteFee: Balance = 10 * MICROUNIT;
+	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_div(4);
+	pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT.saturating_div(4);
+	pub const RelayOrigin: AggregateMessageOrigin = AggregateMessageOrigin::Parent;
+	pub const RelayNetwork: NetworkId = NetworkId::ByGenesis(RELAY_GENESIS);
+	pub RelayChainOrigin: RuntimeOrigin = cumulus_pallet_xcm::Origin::Relay.into();
+	pub UniversalLocation: InteriorLocation = [GlobalConsensus(RelayNetwork::get()), Parachain(ParachainInfo::parachain_id().into())].into();
```

Comment:
Q: why do we need `Parachain(ParachainInfo::parachain_id().into())` as a valid option in universal location?

Reply:
Recommend this be moved into a comment above line 26